### ISSUE TITLE
Ikon for Localizoom

### ISSUE THREAD

**Issue by Majpuc** [Neural-Systems-at-UIO/VisuAlign#151]
Hi, We need an icon for Localizoom viewer as service on EBRAINS. E.g. Siibra explorer has one

**Comment by ieaasebo** [comment]
Yes, great, and just for context - there are implementations underway to be able to add this icon to the landing pages of the datasets for each localizoom link. And/or a preview/thumb image representing the localizoom content potentially. 
E.g. like the siibra: https://raw.githubusercontent.com/FZJ-INM1-BDA/siibra-explorer/master/docs/images/siibra-explorer-square.jpeg
